utils/log-collector

```diff
@@ -54,48 +54,55 @@ now=$(date +"%F")
 outputFormat="${outputFormat:=text}"
 
 # Create a temporary directory to store our logs
-pwd=$(pwd)
-dir=$(mktemp -d "$pwd"/buildkite-logs-"$now")
+dir=$(mktemp -d buildkite-logs-"$now")
```

Comment:
I've got a separate PR since this breaks on GNU I think. Maybe just roll it into this one. See #1019